### PR TITLE
This gem is bullshit w/o nans and infs

### PR DESCRIPTION
**Pull request by dam13n**

### CODE REVIEW

File: lib/is_bullshit.rb [hunk 1]
```diff
@@ -13,10 +14,12 @@ class Object
     '0' => true,
     'nil' => true,
     'null' => true,
+    Float::INFINITY => true,
+    BigDecimal::INFINITY => true
   }
 
   def bullshit?
-    !!BULLSHIT_VALUES[self]
+    !!BULLSHIT_VALUES[self] or (self.nan? if [Float, BigDecimal].member? self.class)
```
Comment (dudo) [inline lib/is_bullshit.rb:22]: Won't this fail when looking for nil or null? I think you'll need to put that on a different line to allow `!!BULLSHIT_VALUES[self]` to continue to work

Comment (dam13n) [reply lib/is_bullshit.rb:22]: Do you have an example? It works w/ `nil` and passes the tests. Can `null`be even an object in ruby??

**Comment by alexford** [comment]
Can you add these cases to the tests?

**Comment by alexford** [comment]
#10 was a pretty big refactor so closing this

**Pull request closed without merging**